packages/babel-helper-function-name/src/index.ts

```diff
@@ -150,7 +153,10 @@ function visit(node, name, scope) {
  * @param {NodePath} param0
  * @param {Boolean} localBinding whether a name could shadow a self-reference (e.g. converting arrow function)
  */
-export default function ({ node, parent, scope, id }, localBinding = false) {
+export default function (
+  { node, parent, scope, id }: any,
```

Comment:
```suggestion
  { node, parent, scope, id }: { node: any, parent: any, scope: any, id: any },
```

It's still full of `any`s, but at least it gives basic safety against typos and autocompletion.

Reply:
done, 
Ideally it should use `NodePath`, but there is an issue about circular dependency to use it